code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -92,7 +92,7 @@
 
 /datum/species/monkey/handle_mutations_and_radiation(mob/living/carbon/human/H)
 	. = ..()
-	if(H.radiation > RAD_MOB_MUTATE * 2 && prob(50))
+	if(H.radiation > RAD_MOB_MUTATE * 2 && prob(10))
```

Comment:
doesn't handle_mutations_and_radiation() have an argument for delta_time now? if so, this should be changed to use delta_time.

Reply:
are you talking about the radiation amount or the probability check

Reply:
and I'll try to work on patching the trick that's currently being used to consistently and rapidly irradiate willing subjects later tonight

Reply:
Why do we do this to ourselves? Bad Coiax for not using it before and for not including the full args list in the proc definition.

Reply:
nvm, timber said on discord that the enzyme thing is a feature

and the delta_time thing should be for the probability, not the rad threshold, I think